common/Root.tsx

```diff
@@ -17,6 +17,7 @@ import { Store } from 'redux';
 import { pollOfflineStatus } from 'actions/config';
 import { AppState } from 'reducers';
 import { RouteNotFound } from 'components/RouteNotFound';
+import 'what-input';
```

Comment:
What is this package being used for? 

https://github.com/ten1seven/what-input gives some information, but I don't see why we need the functionality.

Reply:
Looks like it's used in sass/styles.scss to suppress the focus ring. I think the idea is to not show the focus ring if a user is clicking around, but to show it when they're navigating via keyboard. Haven't run the site yet, but I wonder if this may have some unintended consequences. But I like the concept.

Reply:
`what-input` lets you remove the big blue outline around buttons & links only if a user is using the mouse to navigate (so not to remove it for users who need that accessibility feature)